src/dwn.ts

```diff
@@ -40,6 +41,7 @@ export class Dwn {
 
     this.methodHandlers = {
       [DwnInterfaceName.Events + DwnMethodName.Get]          : new EventsGetHandler(this.didResolver, this.eventLog),
+      [DwnInterfaceName.Messages + DwnMethodName.Get]        : new MessagesGetHandler(this.didResolver, this.messageStore, this.dataStore),
```

Comment:
Sanity: no explicit handler method? Was there more discussions on this while I was away?

Reply:
nothing to the contrary was discussed while you were gone. went ahead and added the explicit `handleMessagesGet` method to `dwn`